org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
```

Comment:
does it really support wayland? if yes then it is the `fallback-x11` socket above instead of `x11`

Reply:
Yes, it really supports Wayland. I changed to `fallback-x11`, thanks for the hint.